### ISSUE TITLE
numpy-1.16 causes failures in kohonen mechanism.

### ISSUE THREAD

**Issue by jvesely** [PrincetonUniversity/PsyNeuLink#1044]
https://travis-ci.org/PrincetonUniversity/PsyNeuLink/jobs/483269995

**Comment by jvesely** [comment]
workarounds (restrict numpy version):
devel: https://github.com/PrincetonUniversity/PsyNeuLink/pull/1036
master: https://github.com/PrincetonUniversity/PsyNeuLink/pull/1037

**Comment by jvesely** [comment]
Probably related to `numpy.matrix` reprecation

**Comment by jvesely** [comment]
The error:
```
self = (Kohonen Kohonen Learning Function-83)
variable = array([[0, 0],
       [0, 0],
       [list([0, 0]), list([0, 0])]], dtype=object)
context = <ContextFlags.COMPONENT: 4096>
    def _validate_variable(self, variable, context=None):
        variable = super()._validate_variable(variable, context)
    
        # variable = np.squeeze(np.array(variable))
    
        name = self.name
        if self.owner and self.owner.name:
            name = name + " for {}".format(self.owner.name)
    
        if not is_numeric(variable):
            raise ComponentError("Variable for {} ({}) contains non-numeric entries".
                                 format(name, variable))
    
        if len(variable)!=3:
            raise FunctionError("variable for {} has {} items ({}) but must have three:  "
                                "input pattern (1d array), activity array (1d array) and matrix (2d array)"
                                "".format(name, len(variable), variable))
    
        input = np.array(variable[0])
        activity = np.array(variable[1])
        matrix = np.array(variable[2])
    
        if input.ndim != 1:
            raise FunctionError("First item of variable ({}) for {} must be a 1d array".
                                format(input, name))
    
        if activity.ndim != 1:
            raise FunctionError("Second item of variable ({}) for {} must be a 1d array".
                                format(activity, name))
    
        if matrix.ndim != 2:
            raise FunctionError("Third item of variable ({}) for {} must be a 2d array or matrix".
>                               format(matrix, name))
E           psyneulink.core.components.functions.function.FunctionError: 'Third item of variable ([list([0, 0]) list([0, 0])]) for Kohonen Learning Function-83 must be a 2d array or matrix'
```

**Comment by jvesely** [comment]
This was fixed/worked around in #1342
